test/services/concerns/xccdf/rules_test.rb

```diff
@@ -33,12 +33,19 @@ def initialize(report_contents)
     end
   end
 
-  test 'returns rules saved in the report' do
+  test 'returns only rules saved with the report' do
+    original_rule = ::Rule.from_openscap_parser(@mock.op_rules.sample)
```

Comment:
The original test here does not match the behavior, previously all rules would be instantiated and saved in `@rules`, now only new ones are.